### ISSUE TITLE
Db setup issues

### ISSUE THREAD

**Issue by resixh** [pvyParts/allianceauth-corp-tools#17]
(auth) root@Alliance-Auth:/home/allianceserver# python /home/allianceserver/myauth/manage.py setup_corptools_models
Traceback (most recent call last):
  File "/home/allianceserver/myauth/manage.py", line 22, in <module>
    execute_from_command_line(sys.argv)
  File "/home/allianceserver/venv/auth/lib/python3.8/site-packages/django/core/management/__init__.py", line 401, in execute_from_command_line
    utility.execute()
  File "/home/allianceserver/venv/auth/lib/python3.8/site-packages/django/core/management/__init__.py", line 395, in execute
    self.fetch_command(subcommand).run_from_argv(self.argv)
  File "/home/allianceserver/venv/auth/lib/python3.8/site-packages/django/core/management/__init__.py", line 244, in fetch_command
    klass = load_command_class(app_name, subcommand)
  File "/home/allianceserver/venv/auth/lib/python3.8/site-packages/django/core/management/__init__.py", line 37, in load_command_class
    module = import_module('%s.management.commands.%s' % (app_name, name))
  File "/usr/lib/python3.8/importlib/__init__.py", line 127, in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
  File "<frozen importlib._bootstrap>", line 1014, in _gcd_import
  File "<frozen importlib._bootstrap>", line 991, in _find_and_load
  File "<frozen importlib._bootstrap>", line 975, in _find_and_load_unlocked
  File "<frozen importlib._bootstrap>", line 671, in _load_unlocked
  File "<frozen importlib._bootstrap_external>", line 783, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/home/allianceserver/venv/auth/lib/python3.8/site-packages/corptools/management/commands/setup_corptools_models.py", line 3, in <module>
    from corptools.tasks import update_or_create_map, process_ores_from_esi, update_ore_comp_table
  File "/home/allianceserver/venv/auth/lib/python3.8/site-packages/corptools/tasks.py", line 21, in <module>
    from . import views
  File "/home/allianceserver/venv/auth/lib/python3.8/site-packages/corptools/views.py", line 20, in <module>
    from .tasks import update_character, update_all_characters, update_ore_comp_table, update_or_create_map, process_ores_from_esi, update_all_corps, check_account
ImportError: cannot import name 'update_character' from partially initialized module 'corptools.tasks' (most likely due to a circular import) (/home/allianceserver/venv/auth/lib/python3.8/site-packages/corptools/tasks.py)

What do i need to do to get this fixed?

**Comment by dhdavvie** [comment]
A quick fix I found is to edit the following file https://github.com/pvyParts/allianceauth-corp-tools/blob/master/corptools/tasks.py and moving the line `from . import views` to the bottom of the file.

TLDR is that `tasks` import `views` near the start of initialisation, then `views` tries to import something from `tasks` but that is not done initialising so it can't. The import at the end allows the relevant parts of tasks.py to be initialised and got me unblocked at least
